src/elements/common/annotator-context/types.ts

```diff
@@ -24,7 +24,7 @@ export interface Metadata {
 }
 
 export interface AnnotationActionEvent {
-    annotation: object | undefined;
-    error: Error | undefined;
+    annotation?: object;
```

Comment:
Should there be a more precise type for this?

Reply:
Yeah, but I figure I'm waiting for `box-annotations` to be imported so I can use those types